### DISCUSSION TITLE
How to push different grouping keys each to pushgateway in one app?

### DISCUSSION THREAD

**Discussion by namsoo2** [prometheus/pushgateway#496] [Q&A]
I am running a Job on multiple nodes using Spring batch.
And basically, PushGateway's grouping key is used as `host` and `job name`.

And while maintaining the above metrics, 
I want to create a logic that is sent once again by excluding the `host` from the grouping key and configuring it only with the `job name`.

for example
When A Job is executed, we want to have two Groups in PushGateway.
Group1 - `host=node1`, `jobName=A Job`
Group2 - `jobName=A Job`

I wonder if this is possible and how do you think about it?

**Comment by namsoo2** [comment]
I'm sorry.
It seems more appropriate to ask this question on Stack Overflow.

https://stackoverflow.com/questions/73889009/how-to-push-different-grouping-keys-each-to-pushgateway-in-spring-batch

**Comment by beorn7** [comment]
It's also fine to ask here.

You can do what you want, but you have to do it in two separate pushes. It's just another line in your code.
